### DISCUSSION TITLE
ERROR: Content security policy script-src: 'unsafe-eval' required

### DISCUSSION THREAD

**Discussion by alexandrecz** [decaporg/decap-cms#5932] [Q&A] [answered]
Guys, 
Could you help with that:
Which is the right CSP header to add to ensure that the CMS Admin will run properly and Why? 
Also, I didn't found on the Netlify CMS Documentation any instruction regarding that matter.

Thanks in advanced!

**Comment by erezrokah** [comment, marked as answer]
Hi @alexandrecz, we can discuss and track this under https://github.com/netlify/netlify-cms/issues/2138
